src/displayBackend/drm/Display.cpp

```diff
@@ -244,6 +247,96 @@ FrameBufferPtr Display::createFrameBuffer(DisplayBufferPtr displayBuffer,
  * Private
  ******************************************************************************/
 
+bool Display::isDrmDevice(udev_device *device)
```

Comment:
Next two functions are completely non-idiomatic for C++. I'd prefer to rewrite them in C++ style.

Reply:
will re-work